app/assets/stylesheets/legacy/elements_details.css

```diff
@@ -17,25 +17,25 @@
   background: white;
 }
 
-.full-screen .eln-panel-detail > .panel-body {
+.full-screen .detail-card > .panel-body {
```

Comment:
We should be able to remove all `.panel-body` classes because we migrated everything to use `<Card>`.